src/utils.jl

```diff
@@ -39,3 +40,128 @@ corresponding to the number of neurons on the last layer of the NN.
 function outdim(model::Chain)::Number
     return [size(p) for p in Flux.params(model)][end][1]
 end
+
+""" 
```

Comment:
Nice detail here! Just a formatting thing: can you use maths notation that's recognized by Documenter.jl please? See [here](https://github.com/JuliaTrustworthyAI/ConformalPrediction.jl/blob/3eeb3cd1ba6fc0a7afebf74901553395e6e950fd/src/conformal_models/inductive_regression.jl#L21) for an example and [here](https://documenter.juliadocs.org/stable/showcase/#Mathematics) for the docs.

Reply:
i checked the results in the julia repl but i was not able to make the function appear in the  documentation. i tried to go in docs/ and then type julia make.jl and i got   "[ Info: SetupBuildDirectory: setting up build directory.
[ Info: Doctest: running doctests.
[ Info: ExpandTemplates: expanding markdown templates.
[ Info: CrossReferences: building cross-references.
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
[ Info: RenderDocument: rendering document.
[ Info: HTMLWriter: rendering HTML pages.
┌ Warning: Unable to determine the repository root URL for the navbar link.
│ This can happen when a string is passed to the `repo` keyword of `makedocs`.
│ 
│ To remove this warning, either pass a Remotes.Remote object to `repo` to completely
│ specify the remote repository, or explicitly set the remote URL by setting `repolink`
│ via `makedocs(format = HTML(repolink = "..."), ...)`.
└ @ Documenter.HTMLWriter C:\Users\Pasqu\.julia\packages\Documenter\qoyeC\src\html\HTMLWriter.jl:732
[ Info: Automatic `version="0.2.1"` for inventory from ..\Project.toml
┌ Warning: Documenter could not auto-detect the building environment. Skipping deployment.
└ @ Documenter C:\Users\Pasqu\.julia\packages\Documenter\qoyeC\src\deployconfig.jl:76 "
 so i opened index.html with edge through vs code and my functions do not appear in the documentations when i search for them. do i have to modify some sort of flag?